src/ui/public/filter_bar/lib/__tests__/mapMatchAll.js

```diff
@@ -0,0 +1,51 @@
+
+describe('ui/filter_bar/lib', function () {
+  describe('mapMatchAll()', function () {
+    const expect = require('expect.js');
+    const ngMock = require('ngMock');
+    let resolvePromises;
+    let mapMatchAll;
+    let filter;
+
+
+    beforeEach(ngMock.module('kibana'));
+    beforeEach(ngMock.inject(function (Private, $rootScope) {
+      resolvePromises = () => $rootScope.$apply();
```

Comment:
Yeah, I was just trying to keep the tests as expressive as possible, but I'm not married to this approach.  I'll swap it out with direct access to $rootScope.